.github/workflows/cs_check.yml

```diff
@@ -0,0 +1,31 @@
+name: "Coding Style Check"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - 'app'
+      - 'system'
+
+jobs:
+  build:
+    name: PHP ${{ matrix.php-versions }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: ['7.4']
+    steps:
+      - name: Setup PHP Action
+        uses: shivammathur/setup-php@1.8.2
```

Comment:
We should use `setup-php@v2`, unless you have some compelling reason to stick at that version.

Reply:
fixed.